src/modules/email/email.service.ts

```diff
@@ -0,0 +1,54 @@
+import { injectable } from 'inversify';
+import jwt from 'jsonwebtoken';
+import config from '../../config/config.ts';
+import { IUser } from '../user/user.interfaces.ts';
+import nodemailer, { TransportOptions } from 'nodemailer';
+
+export const transport = nodemailer.createTransport(
+  config.email.smtp as TransportOptions,
+);
+
+@injectable()
+export class EmailService {
+  public async createEmailConfirmationToken(user: IUser) {
```

Comment:
It might be better to put this method into user model while this class would be responsible for email sending logic.